src/driver/AmdCompiler.cpp

```diff
@@ -319,7 +320,7 @@ class AMDGPUCompiler : public Compiler {
   void AddCommonArgs(std::vector<const char*>& args);
   void FilterArgs(ArgStringList& args);
   bool InvokeDriver(ArrayRef<const char*> args);
-  bool InvokeLLVMLink(ArrayRef<const char*> args);
+  bool InvokeLinker(ArrayRef<const char*> args, const std::string& sLinker);
```

Comment:
This method serves for both: LLVMLink and ld.lld, they both are actually linkers. Moreover, this is a temporary solution till the problem with embeded ld.lld will be soved. Finally the method will be renamed back to InvokeLLVMLink to be used only in fallback path, or maybe eliminated at all (if we decide to leave only in-process path).